### PR TITLE
[promtail] service-extra: custom labels not working

### LINKED ISSUE #3219: [promtail] service-extra: custom labels not working
As per the values file if I am adding extra services I can add also add custom labels:

For example with the below config I would expect my service to have the `pippo: pluto` label
```
extraPorts:
  syslog:
    name: tcp-syslog
    annotations: {}
    labels:
      pippo: pluto
```
However the service-extra.yaml template and dumps `$` instead of `{{- with $values.labels }}` resulting in a broken Service.

### PR DESCRIPTION
**Pull request by camrossi**
This fixes #3219

### CODE REVIEW

File: charts/promtail/templates/service-extra.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ metadata:
   namespace: {{ include "promtail.namespaceName" $ }}
   labels:
     {{- include "promtail.labels" $ | nindent 4 }}
-    {{- with .labels }}
-    {{- toYaml $ | nindent 4 }}
+    {{- with $values.labels }}
```
Comment (Sheikh-Abubaker) [inline charts/promtail/templates/service-extra.yaml:10]: Have you tested it ?

**Review by Sheikh-Abubaker — COMMENTED** [review]
Have you tested

**Review by Sheikh-Abubaker — COMMENTED** [review]
Hey there @camrossi , could you please bump the chart version to `6.16.4` at [charts/promtail/Chart.yaml](https://github.com/grafana/helm-charts/blob/main/charts/promtail/Chart.yaml#L6C10-L6C16) to fix CI lint test.

File: charts/promtail/templates/service-extra.yaml (hunk 1 above)
Comment (camrossi) [reply charts/promtail/templates/service-extra.yaml:10]: Hi @Sheikh-Abubaker yes I did, here an extract from an environment where I have deployed this:
```
promtail:
  extraPorts:
    Fab1:
      name: fab1
      labels:
        type: bgp-ingress
      containerPort: 1513
      protocol: TCP
      service:
        type: LoadBalancer
        port: 1514
```

```
kgs aci-mon-stack-promtail-fab1 --show-labels
NAME                          TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)          AGE     LABELS
aci-mon-stack-promtail-fab1   LoadBalancer   10.109.206.89   192.168.12.135   1514:30646/TCP   3d17h   app.kubernetes.io/instance=aci-mon-stack,app.kubernetes.io/managed-by=Helm,app.kubernetes.io/name=promtail,app.kubernetes.io/version=3.0.0,helm.sh/chart=promtail-6.16.3-camrossi,type=bgp-ingress
```

See the last label.

Cheers!

**Review by zanhsieh — APPROVED** [review]

File: charts/promtail/templates/service-extra.yaml (hunk 1 above)
Comment (Sheikh-Abubaker) [reply charts/promtail/templates/service-extra.yaml:10]: LGTM!

**Review by Sheikh-Abubaker — APPROVED** [review]
LGTM!!

**Pull request merged**